src/api/EscargotPublic.cpp

```diff
@@ -344,14 +351,14 @@ BooleanObjectRef* PointerValueRef::asBooleanObject()
     return toRef(toImpl(this)->asBooleanObject());
 }
 
-bool PointerValueRef::isRegExpObject(ExecutionState& state)
+bool PointerValueRef::isRegExpObject(ExecutionStateRef* state)
```

Comment:
<from here>
Check pointer-type parameter / check before deferencing

Reply:
I added it